### PR TITLE
REVAI-3854: Usage examples

### PR DESCRIPTION
**Pull request by kirillatrev**
REVAI-3854: Usage examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,33 @@ try {
...
+You can request transcript summary.
+
+```
+String urlLinkToFile = "https://www.rev.ai/FTC_Sample_1.mp3";
+
+RevAiJobOptions revAiJobOptions = new RevAiJobOptions();
+revAiJobOptions.setSourceConfig(urlLinkToFile, null);
+revAiJobOptions.setLanguage("en");
+revAiJobOptions.setSummarizationOptions(new SummarizationOptions().setModel(NlpModel.STANDARD));
+
```
Comment (dmtrrk) [inline README.md]: redundant
upd. the line 81 is the empty line. it does not look like we need it

File: examples/AsyncSummarizeMediaUrl.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      throw new RuntimeException("Failed to submit url [" + mediaUrl + "] " + e.getMessage());
+    }
+    String jobId = submittedJob.getJobId();
+    System.out.println("Job Id: " + jobId);
+    System.out.println("Job Status: " + submittedJob.getJobStatus());
+    System.out.println("Created On: " + submittedJob.getCreatedOn());
+
+    // Waits 5 seconds between each status check to see if job is complete
+    boolean isJobComplete = false;
+    while (!isJobComplete) {
```
Comment (dmtrrk) [inline examples/AsyncSummarizeMediaUrl.java:48]: once this is a summarization example, not sure we need this block as it makes the example more complicated.
can we do a summarization polling only?

Comment (kirillatrev) [reply examples/AsyncSummarizeMediaUrl.java:48]: @alexsku are you okay with that?
You were requesting polling those jobs in order in python sdk...

**Review by dmtrrk — APPROVED** [review]
there is a question regarding polling, but it looks like this approach is ok for SDKs in other languages

**Pull request merged**